airbyte-webapp-e2e-tests/cypress/commands/connector.ts

```diff
@@ -1,18 +1,19 @@
-import { enterDestinationPath, selectServiceType, enterName, enterHost, enterPort, enterDatabase, enterUsername, enterPassword, enterPokemonName } from "pages/createConnectorPage"
+import { enterDestinationPath, selectServiceType, enterName, enterHost, enterPort, enterDatabase, enterUsername, enterPassword, enterPokemonName, enterSchema } from "pages/createConnectorPage"
 
-export const fillPostgresForm = (name: string, host: string, port: string, database: string, username: string, password: string) => {
+export const fillPostgresForm = (name: string, host: string, port: string, database: string, username: string, password: string, schema: string) => {
   cy.intercept("/api/v1/source_definition_specifications/get").as(
     "getSourceSpecifications"
   );
   
   selectServiceType("Postgres");
-  
-  cy.wait("@getSourceSpecifications");
-  
+    
   enterName(name);
   enterHost(host);
   enterPort(port);
   enterDatabase(database);
+  if (schema != "") {
```

Comment:
```suggestion
  if (schema !== "") {
```
With Lake's linting PR it would likely flag this.